site/content/blog/2025-02-15-portswigger-lab-file-path-traversal-simple-case/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Solving Portswigger Lab File Path Traversal Simple Case with ZAP"
+summary: "Video and explanation of How to Solve the Portswigger labs using ZAP, in this case: 'Path Traversal Simple Case'"
+date: 2025-02-15
```

Comment:
Can hopefully get this published tomorrow, so change the directory and date to 2025-02-27 ?